modules/claims_api/lib/claims_api/v2/disability_compensation_pdf_mapper.rb

```diff
@@ -9,21 +9,39 @@ def initialize(auto_claim, pdf_data)
       end
 
       def map_claim
+        claim_attributes
+        veteran_info
+
+        @pdf_data
+      end
+
+      def claim_attributes
+        @pdf_data[:data][:attributes] = @auto_claim.deep_symbolize_keys
         claim_date
-        claim_process_type
+
         @pdf_data
       end
 
       def claim_date
-        @pdf_data[:data][:attributes][:claimCertificationAndSignature][:dateSigned] =
-          @auto_claim['claimDate']
+        @pdf_data[:data][:attributes].merge!(claimCertificationAndSignature: { dateSigned: @auto_claim['claimDate'] })
+        @pdf_data[:data][:attributes].delete(:claimDate)
         @pdf_data
       end
 
-      def claim_process_type
-        @pdf_data[:data][:attributes][:claimProcessType] = @auto_claim['claimProcessType']
+      def veteran_info
+        @pdf_data[:data][:attributes].merge!(
+          identificationInformation: @auto_claim['veteranIdentification'].deep_symbolize_keys
+        )
+        zip
+
         @pdf_data
       end
+
+      def zip
+        zip = @auto_claim['veteranIdentification']['mailingAddress']['zipFirstFive'] +
```

Comment:
Is there any chance the data would come back as a number instead of a string? Is there any way this would math instead of concat?